sample-notificator-domain/src/main/groovy/life/qbic/business/notification/send/SendEmail.java

```diff
@@ -0,0 +1,51 @@
+package life.qbic.business.notification.send;
+
+import java.util.List;
+import java.util.function.Consumer;
+import life.qbic.business.notification.create.NotificationContent;
+
+/**
+ * <p>Implements the business logic of sending emails for project updates.</p>
+ */
+public class SendEmail implements Consumer<List<NotificationContent>>, SendEmailInput{
+    private final EmailSender<NotificationEmail> sendOrRemember;
+    private final FailureEmailSender informAdmin;
```

Comment:
One small comment again on the naming. What do you think of renaming like this:

```suggestion
    private final AdminEmailSender informAdmin;
```
```suggestion
    private final FailureEmailSender failureEmail;
```
And the same then also in the DependencyManager. That would reduce the complexity and makes it less confusing.